gui/launcher.cpp

```diff
@@ -238,29 +238,21 @@ void LauncherDialog::build() {
 #endif
 	if (!g_system->hasFeature(OSystem::kFeatureNoQuit))
 		new ButtonWidget(this, _title + ".QuitButton", _("~Q~uit"), _("Quit ScummVM"), kQuitCmd);
+
 	new ButtonWidget(this, _title + ".AboutButton", _("A~b~out"), _("About ScummVM"), kAboutCmd);
-	if (g_system->getOverlayWidth() > 320)
-		new ButtonWidget(this, _title + ".OptionsButton", _("Global ~O~ptions..."), _("Change global ScummVM options"), kOptionsCmd);
-	else
-		new ButtonWidget(this, _title + ".OptionsButton", _c("Global ~O~pts...", "lowres"), _("Change global ScummVM options"), kOptionsCmd);
+	new ButtonWidget(this, _title + ".OptionsButton", _("Global ~O~ptions..."), _("Change global ScummVM options"), kOptionsCmd, 0, _c("Global Opts...", "lowres"));
 
 	// Above the lowest button rows: two more buttons (directly below the list box)
+	DropdownButtonWidget *addButton =
+		new DropdownButtonWidget(this, _title + ".AddGameButton", _("~A~dd Game..."), _("Add games to the list"), kAddGameCmd);
+	addButton->setLowresLabel(_c("~A~dd Game...", "lowres"));
```

Comment:
Can't we have the same interface for DropdownButtonWidget() as we do with ButtonWidget()? i.e. a new optional parameter for lowres. Since lowres labels are set during class instantiation, I see no reason to have a separate setter

Reply:
fixed